tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Copyright (c) IBM Corporation 2019, 2020, 2023
+# Copyright (c) IBM Corporation 2019 - 2024
```

Comment:
seems like we've been preferring to use commas for copyright: `2019, 2024`

Reply:
done